.github/workflows/build.yml

```diff
@@ -68,8 +68,15 @@ jobs:
       working-directory: /Users/runner/go/src/github.com/rook/rook
       run: tests/scripts/validate_modified_files.sh gen-rbac
 
-    - name: setup tmate session for debugging
+    - name: consider debugging
       if: failure()
+      run: |
+        if [ "$GITHUB_REPOSITORY_OWNER" = "rook" ] || [ -n "${{ secrets.USE_WEBSHELL }}" ] || [ "$GITHUB_RUN_ATTEMPT" -gt 1 ]; then
```

Comment:
Where would `USE_WEBSHELL` be set? Or if this is a var we're just defining, how about just calling it the `USE_TMATE` secret? And how about a description such as this?
```suggestion
        # Enable tmate only in the Rook fork, where the USE_TMATE secret is set in the repo, or if the action is re-run
        if [ "$GITHUB_REPOSITORY_OWNER" = "rook" ] || [ -n "${{ secrets.USE_TMATE }}" ] || [ "$GITHUB_RUN_ATTEMPT" -gt 1 ]; then
```

Reply:
I was on the fence between `webshell` and `tmate`. I'm totally happy to take your changes.

https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository

Here's an example url:
https://github.com/check-spelling/rook/settings/secrets/actions

<details><summary>Before</summary>

<img width="785" alt="image" src="https://user-images.githubusercontent.com/2119212/176045639-11306095-c96e-448a-b668-fef5d12c1688.png">
</details>

<details><summary>Creating a secret</summary>

https://github.com/check-spelling/rook/settings/secrets/actions/new

<img width="788" alt="image" src="https://user-images.githubusercontent.com/2119212/176045818-82d3efe9-ef05-414e-bbb4-fecf5d3d1414.png">

</details>

<details><summary>With secret</summary>

https://github.com/check-spelling/rook/settings/secrets/actions

<img width="786" alt="image" src="https://user-images.githubusercontent.com/2119212/176045859-7647f095-3de9-4bee-8fbc-de90e34d6dd0.png">

</details>